src/pyscript/__init__.py

```diff
@@ -1,19 +1,20 @@
 """A CLI for PyScript!"""
+import json
 from pathlib import Path
 
 import platformdirs
-import toml
 from rich.console import Console
 
 APPNAME = "pyscript"
 APPAUTHOR = "python"
 
 
 DATA_DIR = Path(platformdirs.user_data_dir(appname=APPNAME, appauthor=APPAUTHOR))
-CONFIG_FILE = DATA_DIR / Path("pyscript.toml")
-if not DATA_DIR.exists():
-    DATA_DIR.mkdir(parents=True)
-    CONFIG_FILE.touch(exist_ok=True)
+CONFIG_FILE = DATA_DIR / Path("config.json")
```

Comment:
why the name change here?

Reply:
This is a configuration file for the `pyscript` command itself, not the config filename for a PyScript project.